openrtb_ext/imp_adnuntius.go

```diff
@@ -5,4 +5,5 @@ type ImpExtAdnunitus struct {
 	Network   string `json:"network"`
 	NoCookies bool   `json:"noCookies"`
 	MaxDeals  int    `json:"maxDeals"`
+	PriceType string `json:"priceType,omitempty"`
```

Comment:
update [params tests](https://github.com/prebid/prebid-server/blob/master/adapters/adnuntius/params_test.go) to add valid and invalid tests for `PriceType`